packages/mrml-core/src/mj_carousel/render.rs

```diff
@@ -246,10 +263,10 @@ impl<'root> Renderer<'root, MjCarousel, MjCarouselExtra> {
     }
 
     fn render_style(&self) -> Option<String> {
-        if self.element.children.is_empty() {
+        if self.get_children().is_empty() {
             return None;
         }
-        let length = self.element.children.len();
+        let length = self.get_children().len();
```

Comment:
We should create a `children_len` that would lookup the fragment

Reply:
I'm using `count()` on the iterator now, although I'm not sure if there's a better way to do it?